pkgs/development/python-modules/openapi-codec/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+
+# runtime
+, coreapi
+}:
+
+buildPythonPackage rec {
+  pname = "openapi-codec";
+  version = "1.3.2";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-G85jKJ7fU8YB6jaDEgZBQH/2twiAO4lUyKh2/neNIUU=";
+  };
+
+  nativeBuildInputs = [
```

Comment:
Is this line `nativeBuildInputs` required?

Reply:
Removed.